applications/FluidDynamicsApplication/custom_strategies/schemes/bdf2_turbulent_scheme.h

```diff
@@ -62,14 +60,14 @@ namespace Kratos
 /**
  */
 template<class TSparseSpace,class TDenseSpace>
-class GearScheme : public Scheme<TSparseSpace, TDenseSpace>
+class BDF2TurbulentScheme : public Scheme<TSparseSpace, TDenseSpace>
```

Comment:
is it necessary to have `Turbulent` in the name?
Seems a bit superfluous to me

Reply:
I think it is. Not all the schemes allow the use of a turbulent model.